app/views/steps/_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= form_with model: [routine, step], local: true do |form| %>
+  <% if step.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(step.errors.count, "error") %> prohibited this step from being saved:</h2>
```

Comment:
Add test: error msg if not saved

Reply:
Won't test - HTML 5 generates error message that doesn't allow the form to be submitted